### DISCUSSION TITLE
formula for PHP 7.4 is removed ?

### DISCUSSION THREAD

**Discussion by piotrp321** [Homebrew/discussions#178] [Everyday usage] [answered]
Hello 

Formula for php 7.4 was removed ? I trying install and formula is not found. This formula used to work in the past

**Comment by gromgit** [comment]
`php` was upgraded to v8.0, so a new `php@7.4` formula is in progress: https://github.com/Homebrew/homebrew-core/pull/65697

**Comment by hyuraku** [comment, marked as answer]
Now `php@7.4` is here https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/php@7.4.rb
